src/components/RowLink/RowLink.scss

```diff
@@ -0,0 +1,26 @@
+@use '../variables';
+
+$block: '.#{variables.$ns}link-table-utils';
+
+#{$block} {
+    &__main-link-cell::before {
```

Comment:
It seems all these styles should go to the block `row-link` instead of `row-link__main-link-cell`